apps/frontend/src/components/form/GrantForm.tsx

```diff
@@ -278,6 +310,15 @@ const ExampleForm: React.FC<ExampleFormProps> = ({ onSubmit }) => (
         </FormControl>
       )}
     </Field>
+    </Box>
+    <Box py="2">
+    <Text as="i" fontSize={"xs"}>
```

Comment:
⛏️ This isn't indented properly, did you run a linter?

Reply:
Sorry I didn't used linter earlier. Fixed!

Reply:
wait nvm theres a merge conflict let me fix that

Reply:
fixed